scripts/github_api

```diff
@@ -123,6 +157,8 @@ class PullRequest:
 
         pr_info = request('GET', endpoint)
 
+        self.reference = os.path.basename(pr_info['statuses_url'])
```

Comment:
- Not clear to me what you named this "reference" instead of "statuses_url"
- I was expecting to see a variable in the PullRequest object which keeps its statuses. Why there isn't one?